examples/notebooks/beam-ml/mltransform/scale_data.ipynb

```diff
@@ -0,0 +1,566 @@
+{
+  "nbformat": 4,
```

Comment:
For this notebook, I explained the output for `ScaleTo01` since other transforms follow more or less the similar approach.